provider/aws/formation/app.json.tmpl

```diff
@@ -137,6 +148,50 @@
   {{ end }}
 {{ end }}
 
+{{ define "caches" }}
+  {{ with .Manifest.Caches }}
+  "CacheSubnetGroup" : {
```

Comment:
I thought of that. I can see it going either way. Believe that would change things a bit. Perhaps moving the resource to the rack and keeping track of the DB # under the app's stack. Have to look it up, Redis db's are by number and not names(?)